etc/WMAgentConfig.py

```diff
@@ -361,6 +361,7 @@
 config.RucioInjector.RSEPostfix = False  # enable it to append _Test to the RSE names
 config.RucioInjector.metaDIDProject = "Production"
 config.RucioInjector.containerDiskRuleParams = {"weight": "ddm_quota", "copies": 2, "grouping": "DATASET"}
+config.RucioInjector.datasetRuleParams = {}
```

Comment:
Can you please rename it to `blockRuleParams`? The dataset word is extremely confusing when it comes to Rucio vs CMS.

Reply:
Sure, I tried to follow the same convention as the previous lines, but indeed, is extremely confusing when it comes to Rucio-CMS